codec_specs.md

```diff
@@ -182,6 +182,26 @@ Description: The Matroska video stream will contain a demuxed Elementary Stream
 
 Initialization: none
 
+### V_MPEG4/ISO/AVC
+
+Codec ID: V_MPEG4/ISO/AVC
+
+Codec Name: AVC/H.264
+
+Description: Individual pictures of AVC/H.264 stored as described in ISO/IEC 14496-15.
```

Comment:
Does the reference like `[@!ISO.14496.15]` work when you generate the file ?
Also since that's more ISOBMFF related, shouldn't there be a like to the H264 specs ? Like `ITU-T Rec. H.264` or even `[@!ITU.REC.H264]` if possible. The list of "tags" that can be used as reference is [in here](https://xml2rfc.tools.ietf.org/public/rfc) but kinda hard to find something.

Reply:
> Also since that's more ISOBMFF related, shouldn't there be a like to the H264 specs ?

H.264 specs don't define the "file format" used in Matroska, only the "byte stream format", and ISO/IEC 14496-15 defines the "file format" used by Matroska. If you use H.264 specs only the individual frames are not decodable (the "file format" used is not described). So IMO reference to H.264 directly is not relevant here (Matroska references ISO/IEC 14496-15 which references the needed H.264 parts i.e. without the "byte stream format" stuff).

I'll try to used `[@!ISO.14496.15]` method.